### PR TITLE
Bump dotnet auto version

### PR DESCRIPTION
**Pull request by dszmigielski**

### CODE REVIEW

**Comment by Kielek** [comment]
I would enable mem-profiling by default here: https://github.com/search?q=repo%3Asignalfx%2Fmicroservices-demo%20SIGNALFX_PROFILER_ENABLED&type=code

**Review by Kielek — APPROVED** [review]

**Review by lachmatt — APPROVED** [review]

File: src/cartservice/Dockerfile [hunk 1]
```diff
@@ -36,5 +36,6 @@ ENV CORECLR_PROFILER={B4C89B0F-9908-4F73-9F59-0D77C5A06874}
 ENV CORECLR_PROFILER_PATH=/opt/signalfx/SignalFx.Tracing.ClrProfiler.Native.so
 ENV SIGNALFX_DOTNET_TRACER_HOME=/opt/signalfx
 ENV SIGNALFX_PROFILER_ENABLED=true
+SIGNALFX_METRICS_Process_ENABLED=true
```
Comment (Kielek) [inline src/cartservice/Dockerfile]: missing `ENV `?

**Comment by Kielek** [comment]
Could you please check https://github.com/signalfx/microservices-demo/blob/9e2734774f740953b98c0f3bf36e17e433cdc0b6/src/cartservice/tests/docker-compose.yaml#L16 and https://github.com/signalfx/microservices-demo/blob/9e2734774f740953b98c0f3bf36e17e433cdc0b6/kubernetes-manifests/cartservice.yaml#L52 ?

**Review by mzajacsplunk — APPROVED** [review]
Looks good

**Review by emaderer — APPROVED** [review]

**Pull request merged**